promgen/locale/ja/LC_MESSAGES/django.po

```diff
@@ -172,3 +172,11 @@ msgstr "Serviceを登録"
 #: templates/promgen/project_detail_hosts.html:49
 msgid "Silence selected hosts"
 msgstr "選択したHostをSilence"
+
+#: templates/promgen/service_detail.html:52
+msgid "Edit"
+msgstr "Serviceを編集"
```

Comment:
"Edit" is fairly generic, and used by other simple 'edit' buttons, so I'm concerned it will conflict with other edit buttons. We should probably leave this as "Edit Service" in the html templates and it will work with the existing translate strings.

Reply:
Address this in [589e4d8](https://github.com/line/promgen/pull/520/commits/589e4d86e4e9c6258e14e6413218573179431443)